packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -863,6 +864,8 @@ export const prepareVaultManagerKit = (
                 vaultId,
                 'never stored during initVaultKit failure',
               );
+            } finally {
```

Comment:
this seems to be a different try/catch than the one that used to contain `seat.exit()`

Reply:
Good catch! My eye was fooled into doing the wrong curly matching for some reason.

Reply:
Good eyes! Thankfully the tests caught this too